pkg/sync/rpc/server.go

```diff
@@ -155,6 +160,7 @@ func NewSyncAgentServer(startPort, endPort int, replicaAddress, volumeName, inst
 	server := grpc.NewServer(interceptor.WithIdentityValidationReplicaServerInterceptor(volumeName, instanceName))
 	enginerpc.RegisterSyncAgentServiceServer(server, sas)
 	reflection.Register(server)
+	profilerrpc.RegisterProfilerServer(server, profiler.NewServer(volumeName+"-sync-agent"))
```

Comment:
Should we add the suffix `-sync-agent`?

Reply:
In fact, [the name of the server is not used in the implementation](https://github.com/longhorn/go-common-libs/blob/main/profiler/profiler.go#L45-L53). Adding the `-sync-agent` suffix here to avoid potential confusion with the replica's profiler server.